app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+

```

Comment:
[OPTIONAL]
- Here you can add the `after_save :update_comments_counter` like you did for the posts so it can run immediately after the number of comments has updated